packages/plugin-testing-integration/src/app.js

```diff
@@ -151,8 +144,48 @@ async function initImaApp(bootConfigMethods = {}) {
     };
   }
 
-  let app = createImaApp();
-  let bootConfig = getClientBootConfig({
+  /**
+   * Get response content for the application run
+   *
+   * @returns {string} html content
+   */
+  async function _getIMAResponseContent() {
+    // Mock devUtils to override manifest loading
+    const devUtils = {
+      manifestRequire: () => ({
+        ima: {
+          createImaApp,
```

Comment:
This method seems to be called only [here](https://github.com/seznam/ima/blob/4952b4bade955eb0f1e98bf01b31bcffc38a2f9e/packages/server/lib/factory/IMAInternalFactory.js#L82), can tests really get there? I feel like tests are never creating the dummy app, since there is no server-side rendering (at least not right now). So maybe this specific mock can be removed.